src/Fieldtypes/Bard/Augmentor.php

```diff
@@ -181,4 +154,27 @@ protected function augmentSets($value, $shallow)
             return array_merge($values, ['type' => $set['type']]);
         })->all();
     }
+
+    public function renderHtmlToProsemirror(string $value)
+    {
+        return (new Editor(['extensions' => $this->extensions()]))->setContent($value)->getDocument();
+    }
+
+    public function renderProsemirrorToHtml(array $value)
+    {
+        return (new Editor(['extensions' => $this->extensions()]))->setContent($value)->getHTML();
+    }
+
+    public function extensions()
+    {
+        $this->addExtension('image', $this->withStatamicImageUrls ? new StatamicImageNode : new ImageNode);
+
+        foreach (self::$replaceExtensions as $name => $replacement) {
+            self::$extensions[$name] = $replacement;
+        }
+
+        Arr::removeNullValues(self::$extensions);
```

Comment:
I think this needs to be `self::$extensions = Arr::removeNullValues(self::$extensions);`

Reply:
Fixed